projects/admin/src/app/record/brief-view/circ-policies-brief-view.component.ts

```diff
@@ -30,7 +30,7 @@ import { ResultItem } from '@rero/ng-core';
   `,
   styles: []
 })
-export class CircPoliciesBriefViewComponent implements ResultItem {
+export class CircPoliciesBriefViewComponent {
 
   @Input()
```

Comment:
@Input is not mandatory, because record is not passed to component on initialization, but via viewContainerRef.